helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/listener/HelpDeskRoomEventListener.java

```diff
@@ -115,8 +132,30 @@ public void onSymUserLeftRoom(SymUserLeftRoom symUserLeftRoom) {
     SymUser symUser = symUserLeftRoom.getAffectedUser();
     SymStream symStream = symUserLeftRoom.getStream();
 
-    if (isBotUser(symUser) && isAgentStreamId(symStream)) {
-      LOGGER.warn("Bot user was removed from the agent room");
+    if (isAgentStreamId(symStream)) {
```

Comment:
You can simplify this logic:

```
if (!isAgentStreamId(symStream) && (symStream.getMembers().size() <= 1)) {
  Ticket ticket = ticketClient.getTicketByServiceStreamId(symStream.getStreamId());
  
  if (ticket != null) {
    // Do stuff
  }
}
```

The log message is no longer required.

Reply:
Fix'd